src/actors/abi/sector.go

```diff
@@ -109,10 +109,11 @@ type OnChainSurprisePoStVerifyInfo struct {
 	RegisteredProof
 	Candidates []PoStCandidate
 	Proofs     []PoStProof
-	CommT      ChallengeTicketsCommitment
+	// CommT      ChallengeTicketsCommitment // Potential future optimization.
 }
 
 type PoStCandidate struct {
+	RegisteredProof
```

Comment:
Either this one or the one in PrivatePoStCandidateProof is redundant. I'll land it like this, but lmk which one we should remove.